package.json

```diff
@@ -11,7 +11,8 @@
     "travis-deps": "pip install -t lib -r requirements.travis.txt --upgrade",
     "dev-deps": "python -m pip install --no-deps -r requirements.dev.txt",
     "test": "curl -X POST 'http://localhost:15606/reset' && python -m unittest discover -p '*_test.py'  -b",
-    "emulator" : "gcloud beta emulators datastore start --host-port=:15606 --no-store-on-disk --consistency=1.0",
+    "start-emulator" : "gcloud beta emulators datastore start --host-port=:15606 --no-store-on-disk --consistency=1.0",
+    "stop-emulator" : "curl -X POST 'http://localhost:15606/shutdown'",
```

Comment:
These targets are for starting and stopping the emulator. The steps to be executed for running the tests is

1. `npm run start-emulator` 
2. `npm test` - reset the emulator and start runnning the tests. 
3. `npm run stop-emulator`